src/documentation/pages/components/transfer-list/examples/_base.vue

```diff
@@ -0,0 +1,180 @@
+<template lang="pug">
+ComponentExample(title="Base" id="base" padding="-p--0" :tabs="exampleTabs")
```

Comment:
Remove this padding-0, the example is not looking good:

![image](https://github.com/CenturyLink/Chi/assets/148064753/7eb1c8ea-3d65-43cb-aed0-04f39aafc1b3)

Reply:
Indeed. Fixed.